src/pages/AddLiquidity/index.tsx

```diff
@@ -548,6 +550,11 @@ export default function AddLiquidity() {
     }),
     [usdcValueCurrencyB]
   )
+
+  const owner = useSingleCallResult(tokenId ? positionManager : null, 'ownerOf', [tokenId]).result?.[0]
+  const ownsNFT = owner === account || existingPositionDetails?.operator === account
```

Comment:
worth maybe doing lowercase comparison here in case one of these doesn't come back as checksum format? do we have a util for comparing addresses maybe :3